doc/support_matrix.yml

```
@@ -472,6 +487,16 @@ boards:
       wifi: not_available
       ethernet_over_usb: supported
 
+  st-nucleo-wba55:
+    name: ST NUCLEO-WBA55CG
+    url: https://web.archive.org/web/20240803070523/https://www.st.com/en/evaluation-tools/nucleo-wba55cg.html
+    chip: stm32wba55cg
+    tier: "3"
+    support:
+      user_usb: supported
```

That one also needs to be moved to the chip, because the WBA chips do not support USB at all:
```suggestion
      user_usb: not_available
```